fieldservice_available_worker/wizards/fsm_available_worker.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2023 - TODAY, Marcel Savegnago <marcel.savegnago@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+# -*- coding: utf-8 -*-
+from odoo import _, fields, models
+
+
+class FsmAvailableWorker(models.TransientModel):
+    _name = "fsm.available.worker"
+
```

Comment:
Tratar o warning : 2023-11-10 16:26:29,850 26 WARNING devel odoo.models: The model fsm.available.worker has no _description